tensorflow/c/experimental/ops/array_ops.h

```diff
@@ -31,6 +31,10 @@ Status ZerosLike(AbstractContext* ctx,
                  absl::Span<AbstractTensorHandle* const> inputs,
                  absl::Span<AbstractTensorHandle*> outputs, const char* name);
 
+Status Shape(AbstractContext* ctx,
```

Comment:
Is this tested?

Reply:
I was originally using the shape and prod ops to get the number of elements in an AbstractTensorHandle, until I realize that `TF_NumElems` was available for TF_Tensors, I can remove both for now.